content/en/hub/initial-setup/networking/OpenVPN.md

```diff
@@ -0,0 +1,97 @@
+
+* *NET30*: Use one `/30` subnet per client in a point-to-point topology. Designed for use when connecting clients are Windows systems.
+* *P2P*: Point-to-point topology that points the local server and remote client endpoints to each other. Each client is given one IP address. This is only recommmended when none of the clients are a Windows system.
+* *SUBNET*: the interface uses an IP address and subnet. Each client is given one IP address. Windows clients require the **TAP-Win32 driver** version 8.2 or newer. *TAP* devices always use the *SUBNET* **Topology**.
+
+The **Topology** selection is automatically applied to any connected clients.
+
+When **TLS Crypt Auth Enabled** is set, TrueNAS generates a static key for the **TLS Crypt Auth** field after saving the options.
+To change this key, click **RENEW STATIC KEY**.
+This key is required for any clients that will connect to the server.
+It is recommended to back up the key in a secure location so that it can be provided to any client systems that will connect to this server.
+
+Finally, review the [Security Options](#security-options) and choose settings that meet your network security requirements.
+
+After configuring and saving your OpenVPN Server, you can generate client configuration files for importing to any OpenVPN client systems that will connect to this server.
+You will need the certificate from the client system already imported on the system.
+To generate the configuration file, click **DOWNLOAD CLIENT CONFIG** and select the **Client Certificate**.
+
+## OpenVPN Client
+
+Go to the **Services** page, find the **OpenVPN Client** row, and click <i class="material-icons" aria-hidden="true" title="Configure">create</i> (Configure) to set up a TrueNAS OpenVPN Client.
+
+<img src="/images/OpenVPNClientServiceConfig.png">
+<br><br>
+
+Choose the certificate to use as an OpenVPN client.
+This certificate must exist in TrueNAS and be in an active (unrevoked) state.
+Enter the host name or IP address of the **Remote** OpenVPN server.
+
+Continue to review and choose any other [Connection Settings](#connection-settings) that fit with your network environment and performance requirements.
+The **Device Type** must match with the OpenVPN server **Device Type**.
+**Nobind** prevents using a static IP address and port for the client. By default, this is enabled to allow the OpenVPN client and server to run concurrently.
+
+Finally, review the [Security Options](#security-options) and choose settings that meet your network security requirements.
+When the OpenVPN server is using TLS Encryption, copy the static TLS encryption key and paste into the **TLS Crypt Auth** field.
+
+## Service Activation
+
+When you're finished configuring the Server or Client service, click **SAVE**.
+Activate the service by clicking the *Running* toggle.
+Setting **Start Automatically** means the service starts whenever the TrueNAS system is powered on and begins to boot.
```

Comment:
I think we should not say `begins to boot`, it actually starts when the system is considered in `ready` state which is when the boot process is complete with network / pool etc configured and running.